### PR TITLE
DR-2683 Design system upgrade 2.1.3

### PR DESCRIPTION
**Pull request by 7emansell**
## Ticket:

Fixes JIRA ticket [DR-2683](https://jira.nypl.org/browse/DR-2683).

## This PR does the following:

- Updates subtitles for swimlanes and "Explore further" cards (https://jira.nypl.org/browse/DSD-1628)
- Adds lazy loading for card image optimization on swimlanes and "Explore further" cards (https://jira.nypl.org/browse/DSD-1630)
- Changes tests to address lazy loading
- Updates DC logo color (no code change necessary) (https://jira.nypl.org/browse/DSD-1590).
- Corrects FeaturedContent alignment (https://jira.nypl.org/browse/DSD-1633)

### Checklist:

- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by 7emansell** [comment]
Noting that these visual changes are only possible with footer removed, we are waiting on the updated footer to release so it won't override the old styles

**Comment by 7emansell** [comment]
Will be easier to see diff once previous PRs are merged in, sorry

File: src/pages/_document.tsx
Comment (avertrees) [inline src/pages/_document.tsx:10]: did we say we were going to add the footer back?

Comment (7emansell) [reply src/pages/_document.tsx:10]: Yes we did. This won't be merged into main, but it needs to be VQAd like this

File: src/components/featuredContent/featuredContent.tsx
Comment (EdwinGuzman) [inline src/components/featuredContent/featuredContent.tsx:17]: So this an update that the app needs and not something that could go into the component?

File: src/components/swimlanes/swimLanes.tsx
Comment (EdwinGuzman) [inline src/components/swimlanes/swimLanes.tsx:46]: Does it _feel_ faster?...

File: src/components/featuredContent/featuredContent.tsx
Comment (7emansell) [reply src/components/featuredContent/featuredContent.tsx:17]: I guess by Marty not implementing it in the component updates, I assumed it would be. I like this bc I think we should err on the side of design freedom for the text content and I don't think the component necessarily looks bad without this padding– I could see someone not wanting it

File: src/components/swimlanes/swimLanes.tsx
Comment (7emansell) [reply src/components/swimlanes/swimLanes.tsx:46]: LOL

Comment (7emansell) [reply src/components/swimlanes/swimLanes.tsx:46]: Not that I'm noticing but I guess it's a matter of milliseconds anyways

**Review by EdwinGuzman — APPROVED** [review]
Looks good. It's worth asking Clare for another review even though this is mostly visual updates. She pointed out the hgroup issue in a different ticket/pr and I'm wary that this will be flagged again.

**Comment by 7emansell** [comment]
Merging this in with footer in it, so changes will not be visible on main/qa until global footer is updated https://jira.nypl.org/browse/DR-2711

**Pull request merged**
